docs/contrib/PlatformBringUp.md

```diff
@@ -0,0 +1,415 @@
+
+
+```
+# cells_clkgate.v
+##########################
+module OPENROAD_CLKGATE (CK, E, GCK);
+  input CK;
+  input  E;
+  output GCK;
+
+ `ifdef OPENROAD_CLKGATE
+    <clkgate_std_cell> latch (.CLK(CK), .GATE(E), .GCLK(GCK));
+ `else
+    assign GCK = CK;
+ `endif
+endmodule
+```
+
+
+
+## Latch
+
+Next is the generic latch. Once again, this requires a latch standard cell to be used. The structure of this module differs slightly from the `clkgate` module seen previously. Following is the generic structure of the behavioral latch module:
+
+		
+
+
+```
+# cells_latch.v
+########################
+module $_DLATCH_P_(input E, input D, output Q);
+     <d_latch_std_cell> _TECHMAP_REPLACE_ (
+       .D(D),
+       .G(E),
+       .Q(Q)
+       );
+endmodule
+
+module $_DLATCH_N_(input E, input D, output Q);
+     <d_latch_std_cell> _TECHMAP_REPLACE_ (
+       .D(D),
+       .GN(E),
+       .Q(Q)
+       );
+endmodule
+```
+
+
+This file contains two modules, `$_DLATCH_P_` and `$_DLATCH_N_`. Notice that `$_DLATCH_N_` has it’s gate input is negated making it enable on a low signal while `$_DLATCH_P_` has a non-negated gate input making it enable on a high signal.
+
+
+## FastRoute Configuration
+
+FastRoute is the tool used to global-route the design. FastRoute requires a Tcl file to set which routing layers will be used for signals, adjust routing layer resources, set which routing heuristic to use when routing, etc. It’s recommended to use the default `fastroute.tcl` due to its simplicity and effectiveness. Following is the default FastRoute configuration file.
+
+
+```
+# fastroute.tcl
+#####################
+set_global_routing_layer_adjustment $::env(MIN_ROUTING_LAYER)-$::env(MAX_ROUTING_LAYER) 0.5
+
+set_routing_layers -signal $::env(MIN_ROUTING_LAYER)-$::env(MAX_ROUTING_LAYER)
+```
+
+
+The first command, set_global_routing_layer_adjustment, adjusts the routing resources of the design. This effectively reduces the number of routing tracks that the global router assumes to exist. By setting it to the value of 0.5, this reduced the routing resources of all routing layers to 50% which can help with congestion and reduce the challenges for detail routing. The second command, set_routing_layers, sets the minimum and maximum routing layers for signal nets by using the -signal option.
+
+More customization can be done to increase the efficiency of global and detail route. Refer to the [OpenROAD documentation](https://openroad.readthedocs.io/_/downloads/en/latest/pdf/).
+
+
+## Metal Tracks Configuration
+
+OpenROAD requires a metal track configuration file for use in floorplanning. For each metal layer, the x and y offset as well as the x and y pitch are defined. To find the pitch and offset for both x and y, refer to the tech LEF. Following is a generalized metal tracks configuration file with five metal tracks defined.
```

Comment:
Please specific what in the LEF file folks should look at to define the pitch and offset, ie. what in the LEF file helps inform the user.

Reply:
Reworded to be more specific and also added in an example from sky130hd tech file showing the pitch and offset definitions